### PR TITLE
Fix typo in input-httpjson.asciidoc

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
This PR is to add the missing `:` in `input-httpjson.asciidoc`.

### CODE REVIEW

File: x-pack/filebeat/docs/inputs/input-httpjson.asciidoc [hunk 1]
```diff
@@ -33,10 +33,10 @@ filebeat.inputs:
...
   interval: 1m
   request.url: https://api.ipify.org/?format=json
-  processors:
-    - decode_json_fields
-        fields: [message]
-        target: json
+processors:
+- decode_json_fields:
+    fields: ["message"]
+    target: "json"
```
Comment (jsoriano) [inline x-pack/filebeat/docs/inputs/input-httpjson.asciidoc:39]: It is actually [valid](https://www.elastic.co/guide/en/beats/filebeat/7.10/defining-processors.html#where-valid) to define processors at the input level. This doc page is about inputs, so I think we should leave this example in the input.
```suggestion
  processors:
    - decode_json_fields:
        fields: ["message"]
        target: "json"
```

Comment (kaiyan-sheng) [reply x-pack/filebeat/docs/inputs/input-httpjson.asciidoc:39]: Ahh ok! Thank you! Will change it back! @jsoriano Do you know how can I fix the lint error in this PR?

Comment (jsoriano) [reply x-pack/filebeat/docs/inputs/input-httpjson.asciidoc:39]: It seems `NOTICE.txt` is outdated in master, it should have been updated in https://github.com/elastic/beats/pull/23484, not sure why it was not detected. Opening a PR to fix this: https://github.com/elastic/beats/pull/23549

Comment (kaiyan-sheng) [reply x-pack/filebeat/docs/inputs/input-httpjson.asciidoc]: Thank you! It's fixed now.

**Review by jsoriano — APPROVED** [review]

**Pull request merged**